client/src/components/Post/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {menuData} from '../Post/menu-data';
+import {Container, Row} from 'reactstrap';
+import './Post.scss';
+import { booleanLiteral, JSXElement } from '@babel/types';
+export class Post extends React.Component<{}, { isMenuOpen: boolean }> {
+    constructor( props: any ) {
+        super(props);
+        this.state = {
+            isMenuOpen: false,
+        };
+    }
+    public handleMenuButtonClick = (): void => {
+        this.setState({ isMenuOpen: ! this.state.isMenuOpen });
+    }
+    public render(): JSX.Element {
+        const { isMenuOpen }: { isMenuOpen: boolean }  = this.state;
+        return(
+        <div>
+                <Container className=' ml-2 dots' onClick={this.handleMenuButtonClick}>
+                    <Row><text className='mt-1'>.</text></Row>
+                    <Row><text className='mt-1'>.</text></Row>
+                    <Row><text className='mt-1'>.</text></Row>
+                </Container>
+            {menuData.length && (
+                <nav className={ `navig ml-3 mt-2 ${ isMenuOpen ? 'show-menu' : ''}` }>
+                <ul className='list-unstyled menu-items'>
```

Comment:
list-unstyled? u didn't stye it?

Reply:
List-unstyled is a bootstrap class which means the same as list-style:none to remove dots in front of list items